src/utils/formatAmount.ts

```diff
@@ -0,0 +1,32 @@
+export function formatAmount(amount: number): string {
```

Comment:
Use build in [formatters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)